client/components/ReviewRating/WriteReview/WriteReview.jsx

```diff
@@ -0,0 +1,195 @@
+import React, { useState, useEffect } from 'react';
+
+import PropTypes from 'prop-types';
+import axios from 'axios';
+
+const WriteReview = ({ isClickAdd, closeWriteReview, id }) => {
+  const [newReview, setNewReview] = useState({
+    rating: 5,
+    summary: '',
+    body: '',
+    recommend: true,
+    name: '',
+    email: '',
+    photos: '',
+    characteristics: {},
+    product_id: id,
+  });
+  const [isValid, setIsValid] = useState(true);
+  const [photoUrl, setPhotoUrl] = useState([]);
+  const [charName, setCharName] = useState([]);
+  const [charId, setCharId] = useState([]);
+  const [charValue, setCharValue] = useState([]);
+
+  useEffect(() => {
+    axios.get(`/reviews/meta/${id}`)
+      .then((res) => {
+        setCharName(Object.entries(res.data.characteristics).map((i) => i[0]));
+        setCharId(Object.entries(res.data.characteristics).map((i) => i[1]).map((i) => i.id));
+        // eslint-disable-next-line max-len
```

Comment:
This is already pretty hard to understand with all the map (i) map (i) etc. Let's not add rules to disable our eslint unless there's no other way around it. If we want to disable the rule, let's discuss and add it to .eslintrc file.

Reply:
I was struggling with using state on the nested object. That is the way I found to put it into an array so I can map it when the component re-render. I broke down the line so it doesn't break the eslint anymore. thanks